server/src/main/resources/templates/web/eastwest/login.html

```diff
@@ -10,6 +10,18 @@ <h1 th:text="${T(org.cloudfoundry.identity.uaa.zone.IdentityZoneHolder).uaa ? 'W
       <p>East/West environment (<a href="https://cloud.gov/docs/apps/govcloud/">what's this?</a>)</p>
     </div>
 
+    <div class="card warning" style=" width: 80%; margin: 0 auto; background-color: #febfc1; color: #ab2a2a; max-width: 800px; padding-top: 5vh; padding-bottom: 5vh; margin-bottom: 20px;">
+      <p class="h3-primary" style=" font-weight: 800; font-size: 22px; color: #ab2a2a;">
```

Comment:
Same questions about fixed-pixel specs and straight-out-of-the-USWDS alternatives.